airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -83,12 +82,12 @@
   private final ExceptionWrappingDatabase database;
   private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseConfigPersistence.class);
 
-  public DatabaseConfigPersistence(final Database database) {
-    this.database = new ExceptionWrappingDatabase(database);
+  public static ConfigPersistence createWithValidation(final Database database) {
```

Comment:
:information_source: I've made this a public static creator method instead for better readability. I didn't make the constructor private, since it's used in a handful of tests still.